api/app/views/mno_enterprise/jpi/v1/app_instances/_resource.json.jbuilder

```diff
@@ -8,6 +8,15 @@ json.oauth_keys_valid app_instance.oauth_keys_valid
 #json.microsoft_trial_url app_instance.microsoft_trial_url
 json.created_at app_instance.created_at
 
+if app_instance.per_user_licence?
+	json.per_user_licence app_instance.per_user_licence
+	json.licences_count app_instance.user_licences.active.count
```

Comment:
As mentioned via email, we only need the user licence count in mnoe.
So I'd suggest to do the count in MnoHub and expose is as a `active_licences_count` field on the `AppInstance` to avoid extra requests.